src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossSummary.scala

```diff
@@ -46,22 +46,29 @@ object TreeLossSummary {
         val tcd2010: Integer = raster.tile.tcd2010.getData(col, row)
         val biomass: Double = raster.tile.biomass.getData(col, row)
 
-
-        // Optionally calculate stocks in carbon pools in 2000
-        val carbonPools: Boolean =
-          getAnyMapValue[Boolean](kwargs, "carbonPools")
-
-        val agc2000: Float = raster.tile.agc2000.getData(col, row)
-        val bgc2000: Float = raster.tile.bgc2000.getData(col, row)
-        val soilCarbon2000: Float = raster.tile.soilCarbon2000.getData(col, row)
-
         val grossCumulAbovegroundRemovalsCo2: Float = raster.tile.grossCumulAbovegroundRemovalsCo2.getData(col, row)
         val grossCumulBelowgroundRemovalsCo2: Float = raster.tile.grossCumulBelowgroundRemovalsCo2.getData(col, row)
         val grossEmissionsCo2eNonCo2: Float = raster.tile.grossEmissionsCo2eNonCo2.getData(col, row)
         val grossEmissionsCo2eCo2Only: Float =  raster.tile.grossEmissionsCo2eCo2Only.getData(col, row)
         val netFluxCo2: Float = raster.tile.netFluxCo2.getData(col, row)
         val fluxModelExtent: Boolean = raster.tile.fluxModelExtent.getData(col, row)
 
+        // Optionally calculate stocks in carbon pools in 2000
+        val agc2000: Double = if (getAnyMapValue[Boolean](kwargs, "carbonPools"))
```

Comment:
It would be nice to have:

val wantCarbonPools = getAnyMapValue[Boolean](kwargs, "carbonPools")

and then use that value in the if statements, to avoid calling that function over and over (and less verbose).

Reply:
Good idea. I'll do that.